### DISCUSSION TITLE
How do I create anchors to link to in myst without having headings?

### DISCUSSION THREAD

**Discussion by ssbarnea** [executablebooks/MyST-Parser#618] [Q&A]
I know that headers do automatically get anchors assigned to them but I need to manually create anchors so I can send user to them using the `page#anchor`.

**Comment by chrisjsewell** [comment]
Heya, well if you want to be very "brute force", then you could always as raw HTML:

```html
<div id="abc">
</div>
```

If you are intending to do this "within" MyST,
then you can assign a target to any block (header, paragraph, list, ...)

```markdown
(mytarget)=
a paragraph

{ref}`some text <mytarget>`
```
